### DISCUSSION TITLE
using pysonDB with Pydroid3 (android)

### DISCUSSION THREAD

**Discussion by JdPaz** [pysonDB/pysonDB#108] [Q&A]
I'm trying to use pysonDB with Pydroid3 on Android. I've an error when I try to open the database with the `db.getDb` command. 
pysonDB throws this exception message: `FileSystem does not appear to support flock use softfilelock instead`

Can I use it on Android? Any suggestions are welcome.
Thanks!

**Comment by Adwaith-Rajesh** [comment]
pysondb uses filelock. I don't think PysonDb is going to work on android.
and I don't know of any JSON based DB that work on android. You might have to look for other DB options.
Thank You.
